app/api/email_utils.py

```diff
@@ -74,3 +74,32 @@ def send_email_verification_message(user_name, email):
     html = render_template('email_confirmation.html', confirm_url=confirm_url, user_name=user_name)
     subject = "Mentorship System - Please confirm your email"
     send_email(email, subject, html)
+
+def send_email_request_notification(reciever_id, sender_id, sender_role, end_date, notes):
+    """Sends a mentorship request message to the specified user.
+    1. send_email() for mentorship_relation/send_request route (mentorship_relation.py file):
```

Comment:
these 3 points don't need to be here, because the code is fairly simple to understand, but i understand its a nice addition for anyone who does not understand this at first. Can you summarise this into a paragraph, instead of 3 points?